datadog_checks_dev/datadog_checks/dev/tooling/templates/configuration/instances/jmx.yaml

```diff
@@ -104,6 +104,12 @@
     will give up on that connection and retry.
   value:
     example: 15000
-    type: number
+    type: number 
+
+- name: collect_default_jvm_metrics
+  description: Allows disabling the collection of default JVM metrics
+  value:
+    example: false
```

Comment:
The example should be the default value, therefore `true`. I believe this is the source of confusion. Also, don't hesitate to call out in the wording.